aws-ecr-replicationconfiguration/aws-ecr-replicationconfiguration.json

```diff
@@ -0,0 +1,125 @@
+{
+    "typeName": "AWS::ECR::ReplicationConfiguration",
+    "description": "The AWS::ECR::ReplicationConfiguration resource configures the replication destinations for an Amazon Elastic Container Registry (Amazon Private ECR). For more information, see https://docs.aws.amazon.com/AmazonECR/latest/userguide/replication.html",
+    "sourceUrl": "https://github.com/aws-cloudformation/aws-cloudformation-resource-providers-ecr.git",
+    "definitions": {
+        "Rules": {
+            "type": "array",
+            "minItems": 0,
+            "maxItems": 1,
+            "items": {
+                "$ref": "#/definitions/ReplicationRule"
+            },
+            "description": "An array of objects representing the replication rules for a replication configuration. A replication configuration may contain only one replication rule but the rule may contain one or more replication destinations.",
+            "required": [
+                "ReplicationRule"
+            ]
+        },
+        "ReplicationRule": {
+            "type": "object",
+            "properties": {
+                "Destinations": {
+                    "$ref": "#/definitions/Destinations"
+                }
+            },
+            "description": "An array of objects representing the details of a replication destination.",
+            "required": [
+                "Destinations"
+            ]
+        },
+        "Destinations": {
+            "type": "array",
+            "minItems": 1,
+            "maxItems": 25,
+            "items": {
+                "$ref": "#/definitions/ReplicationDestination"
+            },
+            "description": "An array of objects representing the details of a replication destination.",
+            "required": [
+                "ReplicationDestination"
+            ]
+        },
+        "ReplicationDestination": {
+            "type": "object",
+            "properties": {
+                "Region": {
+                    "$ref": "#/definitions/Region"
+                },
+                "RegistryId": {
+                    "$ref": "#/definitions/RegistryId"
+                }
+            },
+            "description": "An array of objects representing the details of a replication destination.",
+            "required": [
+                "Region",
+                "RegistryId"
+            ]
+        },
+        "RegistryId": {
+            "type": "string",
+            "description": "The account ID of the destination registry to replicate to.",
+            "pattern": "^[0-9]{12}$"
+        },
+        "Region": {
+            "description": "A Region to replicate to.",
+            "type": "string",
+            "pattern": "[0-z9a-z-]{2,25}"
```

Comment:
`0-z9a-z-` should probably be `0-9a-z-`, right?

Reply:
Yes. This is a typo. Let me correct that.